source/_includes/pup43_platforms_fedora.markdown

```diff
@@ -0,0 +1,7 @@
+We publish official packages and run automated testing on the following versions for Puppet 4.3:
```

Comment:
Hm, this is a little misleading. We don't actually publish puppet 4 packages. It's a part of the puppet-agent package. Given that we're trying to help people move over to the "new mentality" of AIO-land, we might want to rethink this wording.